app/javascript/styles/mastodon/containers.scss

```diff
@@ -110,7 +107,7 @@
     display: block;
     font-size: 32px;
     line-height: 40px;
-    margin-left: 8px;
+    margin-left: 10px;
```

Comment:
All those changes seem unrelated?

Reply:
The account header is displayed above the OAuth prompt. I’m making sure it looks good together.